accounting/rewind.go

```diff
@@ -101,7 +92,7 @@ func AccountAtRound(account models.Account, round uint64, db idb.IndexerDb) (acc
 	}
 	txns, r := db.Transactions(context.Background(), tf)
 	if r < account.Round {
-		err = ConsistencyError{fmt.Sprintf("queried round r: %d < account.Round: %d", r, account.Round)}
+		err = types.MakeConsistencyError(fmt.Sprintf("queried round r: %d < account.Round: %d", r, account.Round))
```

Comment:
given that you've already did the hard work of having this error object, why don't you want to take advantage of it and place the two rounds inside that object and let the object's `Error()` method to format the string ?

Reply:
The line you pointed is actually that "elsewhere". It was there before. Now I use this error for transaction paging.